.jenkins/pytorch/macos-build.sh

```diff
@@ -61,6 +61,12 @@ export IMAGE_COMMIT_TAG=${BUILD_ENVIRONMENT}-${IMAGE_COMMIT_ID}
 
 python setup.py install
 
+# this is a bit hacky, but not too bad. Bundle the test binaries into
+# the installation directory, so they can catch a free ride on the 7z
+# train.
+mkdir -p ${PYTORCH_ENV_DIR}/miniconda3/lib/python3.6/site-packages/torch/test_binaries/build
```

Comment:
I guess the non-hacky way would be to add a flag to our build to create this install directory in the torch package and install the binaries there. Do you think it would be a lot of effort?

Reply:
I can give it a try. It's really annoying to iterate on the macos CI environment though, so if it's a big hassle I may punt on this. I really don't think it's all that bad

Reply:
I've started working on this in a branch here https://github.com/pytorch/pytorch/pull/10136, but I'd like to get this diff pushed in the meantime without blocking on that

Reply:
It doesn't make any sense to write cmake to install test binaries. But maybe what I would appreciate is a little script that handles this, so we can put this in a centralized area. Actually, for other reasons, I'd like this binaries to show up in the torch directory for Linux too (my CircleCI config for PyTorch uses 7z rather than docker push).